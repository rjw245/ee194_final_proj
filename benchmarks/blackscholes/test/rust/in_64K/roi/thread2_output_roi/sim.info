{'cmdline': ['/h/rwood04/ee194/sniper/run-sniper',
             '-d',
             './thread2_output_roi',
             '-n',
             '8',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--roi',
             '--',
             '../../../../rust/bs',
             '2',
             '../../../../inputs/in_64K.txt'],
 'git_revision': '',
 'host': 'dell25',
 'pin_version': 71313,
 'rusage': (38983.431617,
            193.871527,
            140600,
            0,
            0,
            0,
            408711,
            0,
            0,
            0,
            12168,
            0,
            0,
            0,
            35685623,
            30308306),
 'snipercmd': ['bash',
               '-c',
               '/h/rwood04/ee194/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/rwood04/ee194/sniper/lib/pin_sim -c /h/rwood04/ee194/sniper/config/base.cfg --general/total_cores=8 --general/output_dir=/h/rwood04/ee194/ee194_docs/final_project/ee194_final_proj/benchmarks/blackscholes/test/rust/in_64K/roi/thread2_output_roi --config=/h/rwood04/ee194/sniper/config/nehalem.cfg --config=/h/rwood04/ee194/sniper/config/gainestown.cfg --config=/h/rwood04/ee194/sniper/config/nehalem.cfg -g --general/magic=true -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../../../rust/bs 2 ../../../../inputs/in_64K.txt'],
 't_elapsed': 44161.21906590462,
 't_start': 1461850778.144787,
 'tracecmds': [],
 'user': 'rwood04',
 'vmem': 3604815872L}
