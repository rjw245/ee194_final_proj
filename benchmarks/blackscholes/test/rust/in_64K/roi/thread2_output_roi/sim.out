                                     | Core 0      | Core 1      | Core 2      | Core 3      | Core 4      | Core 5      | Core 6      | Core 7     
  Instructions                       |       58758 |  6654311087 |  6654328062 |           0 |           0 |           0 |           0 |           0
  Cycles                             |  5676280601 |  5676280601 |  5676280601 |  5676280601 |  5676280601 |  5676280601 |  5676280601 |  5676280601
  IPC                                |        0.00 |        1.17 |        1.17 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  Time (ns)                          |  2133940076 |  2133940076 |  2133940076 |  2133940076 |  2133940076 |  2133940076 |  2133940076 |  2133940076
  Idle time (ns)                     |  2133894302 |       52771 |     3877001 |  2133940076 |  2133940076 |  2133940076 |  2133940076 |  2133940076
  Idle time (%)                      |      100.0% |        0.0% |        0.2% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0%
Branch predictor stats               |             |             |             |             |             |             |             |            
  num correct                        |        6123 |   234174319 |   235012486 |           0 |           0 |           0 |           0 |           0
  num incorrect                      |         827 |     7242598 |     6404023 |           0 |           0 |           0 |           0 |           0
  misprediction rate                 |      11.90% |       3.00% |       2.65% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
  mpki                               |       14.07 |        1.09 |        0.96 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
TLB Summary                          |             |             |             |             |             |             |             |            
  I-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |        7859 |  1092356919 |  1092359192 |           0 |           0 |           0 |           0 |           0
    num misses                       |          16 |          76 |          76 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       0.20% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.27 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |       25677 |  3825812605 |  3825825004 |           0 |           0 |           0 |           0 |           0
    num misses                       |         132 |       19464 |       19473 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       0.51% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        2.25 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  L2 TLB                             |             |             |             |             |             |             |             |            
    num accesses                     |         148 |       19540 |       19549 |           0 |           0 |           0 |           0 |           0
    num misses                       |          32 |         407 |         524 |           0 |           0 |           0 |           0 |           0
    miss rate                        |      21.62% |       2.08% |       2.68% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.54 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
Cache Summary                        |             |             |             |             |             |             |             |            
  Cache L1-I                         |             |             |             |             |             |             |             |            
    num cache accesses               |        7859 |  1092356919 |  1092359192 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |        1047 |       35649 |       36019 |           0 |           0 |           0 |           0 |           0
    miss rate                        |      13.32% |       0.00% |       0.00% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |       17.82 |        0.01 |        0.01 |         inf |         inf |         inf |         inf |         inf
  Cache L1-D                         |             |             |             |             |             |             |             |            
    num cache accesses               |       25769 |  3825812860 |  3825825246 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |         681 |     1433115 |     1433943 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       2.64% |       0.04% |       0.04% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |       11.59 |        0.22 |        0.22 |         inf |         inf |         inf |         inf |         inf
  Cache L2                           |             |             |             |             |             |             |             |            
    num cache accesses               |        1739 |     1468771 |     1469962 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |        1271 |     1338775 |     1340110 |           0 |           0 |           0 |           0 |           0
    miss rate                        |      73.09% |      91.15% |      91.17% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |       21.63 |        0.20 |        0.20 |         inf |         inf |         inf |         inf |         inf
  Cache L3                           |             |             |             |             |             |             |             |            
    num cache accesses               |        1283 |     1338783 |     1340110 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |         624 |         783 |         535 |           0 |           0 |           0 |           0 |           0
    miss rate                        |      48.64% |       0.06% |       0.04% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |       10.62 |        0.00 |        0.00 |         inf |         inf |         inf |         inf |         inf
DRAM summary                         |             |             |             |             |             |             |             |            
  num dram accesses                  |         965 |           0 |           0 |           0 |         969 |           0 |           0 |           0
  average dram access latency (ns)   |       70.09 |         inf |         inf |         inf |       69.67 |         inf |         inf |         inf
  average dram queueing delay        |       16.66 |        0.00 |        0.00 |        0.00 |       16.24 |        0.00 |        0.00 |        0.00
  average dram bandwidth utilization |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
Coherency Traffic                    |             |             |             |             |             |             |             |            
  num loads from dram                |          56 |         232 |         100 |           0 |           0 |           0 |           0 |           0
  num loads from dram cache          |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num loads from remote cache        |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
